WalletWasabi.Gui/Controls/WalletExplorer/WalletViewModel.cs

```diff
@@ -34,13 +34,17 @@ protected WalletViewModel(Wallet wallet) : base(wallet)
 				Observable.FromEventPattern(Wallet.TransactionProcessor, nameof(Wallet.TransactionProcessor.WalletRelevantTransactionProcessed)).Select(_ => Unit.Default))
 				.Throttle(TimeSpan.FromSeconds(0.1))
 				.Merge(UiConfig.WhenAnyValue(x => x.LurkingWifeMode).Select(_ => Unit.Default))
+				.Merge(Wallet.Synchronizer.WhenAnyValue(x => x.UsdExchangeRate).Select(_ => Unit.Default))
 				.ObserveOn(RxApp.MainThreadScheduler)
 				.Subscribe(_ =>
 				{
 					try
 					{
 						Money balance = Wallet.Coins.TotalAmount();
 						Title = $"{WalletName} ({(UiConfig.LurkingWifeMode ? "#########" : balance.ToString(false, true))} BTC)";
+
+						var dollars = Wallet.Synchronizer.UsdExchangeRate * balance.ToDecimal(MoneyUnit.BTC);
+						TitleTip = UiConfig.LurkingWifeMode ? "### USD" : dollars <= 0 ? "??? USD" : $"{dollars:.00} USD";
```

Comment:
Zero dollar is ??? USD? If the exchange rate it Zero then should it be ??? otherwise your wallet is just empty so it is 0.00 USD.

Reply:
Fixed